salt/kubelet/kubelet-config.jinja

```diff
@@ -12,7 +12,7 @@ authentication:
   x509:
     clientCAFile: "{{ pillar['ssl']['ca_file'] }}"
   webhook:
-    enabled: false
+    enabled: true
```

Comment:
before we merge this, it might be worth to consider also if enabling `webhook` hasn't have security consequences.